app/assets/stylesheets/administrate/components/_container.scss

```diff
@@ -0,0 +1,6 @@
+.container {
+  background-color: $white;
+  box-shadow: 0 2px 4px rgba($grey-7, 0.24), 0 2px 6px rgba($grey-7, 0.12);
```

Comment:
Double checking to see if its worth making a variable for this one since its a bit lengthy